src/System.Management.Automation/singleshell/config/MshSnapinInfo.cs

```diff
@@ -114,16 +113,6 @@ string customPSSnapInType
                 throw PSTraceSource.NewArgumentNullException("psVersion");
             }
 
-#if CORECLR // CustomPSSnapIn Not Supported On CSS.
-            // CustomPSSnapIn derives from System.Configuration.Install, which is not in CoreCLR.
-            if (customPSSnapInType != null)
-            {
-                throw PSTraceSource.NewArgumentException(
-                          "customPSSnapInType",
-                          MshSnapInCmdletResources.CustomPSSnapInNotSupportedInPowerShellCore);
```

Comment:
Can we remove this resource string from the .resx file?

Reply:
Fixed.